File: src/library/Transform/ShadowBees.py
from .Context import Context
from ..Model import SpriteId


def patch_shadow_bees(context: Context) -> None:
    bees = context.sprites[SpriteId.x79_BEE]
    bees.display_allocation = 0
    bees.draw_shadow = True
    bees.statis = True
    bees.boss_damage_sfx = True
    bees.hp = 0x20

File: library/zelda3/features/sprites/shadow_bees.rs
use crate::zelda3::model::SpriteId;
use crate::zelda3::model::Z3Model;

pub(crate) fn patch_shadow_bees(model: &mut Z3Model) {
    if let Some(bee) = model.sprite_settings.get_mut(&SpriteId::x79_BEE) {
        bee.display_allocation = 0;
        bee.draw_shadow = true;
        bee.statis = true;
        bee.boss_damage_sfx = true;
        bee.hp = 0x16;
    }
}
